### ISSUE TITLE
Anzu the Raven God missing

### ISSUE THREAD

**Issue by GeneralCaveman** [HellscreamWoW/Tracker#3771]
http://www.wowhead.com/npc=23035/anzu

While in the heroic version of the dungeon, the boss does not seem to be spawned.

If I had to guess, the dungeon may be still using the old way to summon him (outlined here).

http://wowwiki.wikia.com/wiki/Essence-Infused_Moonstone

**Comment by KraitX** [comment]
Yes confirmed it wants you to place essence infused moon stone on the altar. having difficulty getting any mounts dropped in dungeons in azeroth and outlands
